tests/test_qnode.py

```diff
@@ -1164,6 +1164,7 @@ class TestShots:
     """Unit tests for specifying shots per call."""
 
     # pylint: disable=unexpected-keyword-arg
+    @pytest.mark.xfail(reason="deprecated. To be removed in 0.44")
```

Comment:
Since this is specifically testing shots on call, I would recommend using `with pytest.warn` here instead.

Reply:
https://github.com/PennyLaneAI/pennylane/pull/7906/commits/bc27fd9593e0f5812be85d65027267af247bb6ce